src/silx/math/colormap.py

```diff
@@ -515,11 +522,12 @@ class NormalizeResult(NamedTuple):
 def normalize(
     data: numpy.ndarray,
     norm: str = "linear",
-    autoscale: AutoScaleModeType = "minmax",
-    vmin: float | None = None,
-    vmax: float | None = None,
-    gamma: float = 1.0,
-) -> NormalizeResult:
+    autoscale: str = "minmax",
+    vmin=None,
+    vmax=None,
+    gamma=1.0,
+    saturation: int = 0,
```

Comment:
Saturation could be a float, right?
I'm wondering if we would need a "generic" way to provide additional args (gamma & saturation).
Or maybe force them to be keyword only arguments

Reply:
No more as we have switch it to int to have simpler handling of the `__cacheColormapRange`.

For the second part to be honest I don't have a strong opinion.